### PR TITLE
Add FuturePLC to List of companies using ArgoCD

### PR DESCRIPTION
**Pull request by warmfusion**
We've been using ArgoCD for a few months now, and are using it to deploy onto 5 on prem clusters, 1 cloud cluster and have implemented some of our own code to support hands-off deployment onto review branches into dedicated namespaces.

Using the App of Apps pattern to self-manage argo, the clusters, and the services on those clusters.

Keep up the good work

### CODE REVIEW

**Review by alexec — CHANGES_REQUESTED** [review]
Thank you Toby - a great addition to the list! Do you think you could sign the CLA please?

File: README.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ Organizations below are **officially** using Argo CD. Please send a PR with your
 1. [Ticketmaster](https://ticketmaster.com)
 1. [Yieldlab](https://www.yieldlab.de/)
 1. [Volvo Cars](https://www.volvocars.com/)
+1. [Future PLC](https://www.futureplc.com/)
```
Comment (alexec) [inline README.md]: minor - but I think this list in alphabetical order?

**Review by alexec — DISMISSED** [review]

**Comment by alexec** [comment]
@warmfusion - we'd love to have this merged -  but we need the CLA signed - can you do that please?

**Review by alexmt — APPROVED** [review]
Got a confirmation that docs changes do not require CLA.  LGTM

**Pull request merged**